Source/AwsCommonRuntimeKit/http/HttpClientConnectionManager.swift

```diff
@@ -13,9 +13,7 @@ public class HttpClientConnectionManager {
     public init(options: HttpClientConnectionOptions, allocator: Allocator = defaultAllocator) throws {
         self.options = options
         self.allocator = allocator
```

Comment:
trivial: do we need to bother storing `self.allocator`?

Reply:
Will update it in the http refactor PR.